README.md

```diff
@@ -48,6 +48,14 @@ export BUNDLED_PULL_SECRET_PATH="/tmp/pull_secret.json"
 ./createdisk.sh crc-tmp-install-data
 ```
 
+## Creating container image for bundles
+
+```
+./gen-bundle-image.sh <version> <openshift/okd/podman>
+```
+
+Note: gpg key is needed to create detach signature of the bundle file which added to image.
```

Comment:
`Note: a GPG key is needed to sign the bundles before they are wrapped in a container image` ?